@navikt/aksel-icons/config/figma/index.ts

```diff
@@ -0,0 +1,84 @@
+import { existsSync, mkdirSync, readdirSync, rmSync, writeFileSync } from "fs";
+import { resolve } from "path";
+import { fetchDownloadUrls, fetchIcons } from "./fetch-icons";
+import { resolveName } from "./icon-name";
+import { makeConfig } from "./make-configs";
+
+/* https://www.figma.com/file/wEdyFjCQSBR3U7FvrMbPXa/Core-Icons-Next?node-id=277%3A1221&t=mUJzFvnsceYYXNL5-0 */
+
+const iconFolder = "./icons";
+
+if (!process.env.FIGMA_TOKEN) {
+  throw new Error("FIGMA_TOKEN not set in .env");
+}
+
+main();
+
+async function main() {
+  console.info("Started icon-update from Figma");
+  const icons = await fetchIcons();
+
+  const images = await fetchDownloadUrls(icons.map((x) => x.node_id));
+
+  if (existsSync(iconFolder)) {
+    rmSync(iconFolder, { recursive: true, force: true });
+  }
+
+  mkdirSync(iconFolder);
+
+  console.info(`Downloading ${Object.keys(images).length} icons from Figma...`);
+
+  let counter = 0;
+  for (const [key, value] of Object.entries(images)) {
```

Comment:
Can we find some more describing names for `key` and `value`? Maybe "iconId" and "iconUrl"?